sessions/recap-project-1/quiz-app/README.md

```diff
@@ -12,13 +12,23 @@ able to:
 later project. But make sure you include the answer texts and bookmark buttons in your app. (You
 might want to look up the "hidden" attribute.)
 
+## Overview
+
+In this project we will have 3 pages:
+
+- **Home page:** were we will be able to view **all** questions.
+- **Bookmark page:** were we will be able to **only** view the bookmark questions, the once that you see in the home page as booked.
```

Comment:
Bookmark Page: This is a dedicated space where you can exclusively view the questions you've bookmarked, similar to the ones marked on the home page.